dev/Common/IsWindowsVersion.h

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#ifndef __ISWINDOWSVERSION_H
```

Comment:
VS detects the classic `#ifndef...#define...#endif` pattern and treats it equivalent to `#pragma once` (same perf benefit)

Since `#pragma` behavior isn't guaranteed to be standard and the classic pattern gives the same results it seems the better approach. Plus there are some cases you want to multiply include a header file with different/more behavior (WIL does this).